src/renderer/src/App.svelte

```diff
@@ -413,6 +422,30 @@
   })
 
   setContextClient(client)
+
+  const validateAndSignIn = () => {
+    if (loginData.email == null || loginData.email === '') {
+      showToast('E-mail cannot be empty', 'red')
+      return
+    }
+
+    if (loginData.password == null || loginData.password === '') {
+      showToast('Password cannot be empty', 'red')
+      return
+    }
+
+    client
```

Comment:
Precisa verificar como pegar esse client no service. Dá uma olhada na documentação do urql, no pior dos casos a gente cria um singleton que pode ser inicializado pelo App.svelte. Queria evitar ter que passar como parâmetro do svelte pro service toda vez q for usar.

Reply:
(talvez o svelte tenha algum mecanismo onde possa importar coisas do context fora de arquivos svelte)